themes/_default.yaml

```diff
@@ -113,6 +114,44 @@ badge:
   padding:
     right-left: 8px
     top-bottom: 6px
+ui-button:
```

Comment:
I'm sorry, I missed this — what was the reason to add `ui-button` root, instead of fixing `button`?

Reply:
@asci I also have the same question. This is answer from @froskie: "Because of backward compatibility. Everything related to the actual button component on UI Kit should be `.ui-button`"

Reply:
but it is not backward compatible because those variables are used only in `button.scss` file, which is basically the only file for styles for button